common_helper_yara/yara_scan.py

```diff
@@ -45,12 +57,12 @@ def _parse_yara_output(output):
 
 
 def _split_output_in_rules_and_matches(output):
-    split_regex = re.compile(r'\n*.*\[.*\]\s\/.+\n*')
+    split_regex = re.compile(r'\n*.*\[.*]\s/.+\n*')
     match_blocks = split_regex.split(output)
     while '' in match_blocks:
         match_blocks.remove('')
 
-    rule_regex = re.compile(r'(.*)\s\[(.*)\]\s([\.\.\/]|[\/]|[\.\/])(.+)')
+    rule_regex = re.compile(r'(.*)\s\[(.*)]\s([/]|[./])(.+)')
```

Comment:
We should rewrite this expression for even more reasonable reasons.

Reply:
I tried to guess the intention of the author and I hope it works now